app/src/test/java/uk/ac/ebi/atlas/search/metadata/HighchartsSunburstAdapterTest.java

```diff
@@ -0,0 +1,110 @@
+        var species1OrganismPart1CellType2 = randomAlphabetic(30);
+        var species1OrganismPart1CellType3 = randomAlphabetic(30);
+        var species1OrganismPart2 = randomAlphabetic(10);
+        var species1OrganismPart2CellType1 = randomAlphabetic(30);
+        var species1OrganismPart2ExperimentAccession = generateRandomExperimentAccession();
+
+        when(cellTypeWheelServiceMock.search(metadataSearchTerm, species1.getName()))
+                .thenReturn(
+                        // Species
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(species1.getName()),
+                                species1OrganismPart1ExperimentAccession)),
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(species1.getName()),
+                                species1OrganismPart2ExperimentAccession)),
+                        // Organism part 1
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(species1.getName(), species1OrganismPart1),
+                                species1OrganismPart1ExperimentAccession)),
+                        // Organism part 2
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(species1.getName(), species1OrganismPart2),
+                                species1OrganismPart2ExperimentAccession)),
+                        // Organism part 1, cell types
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(
+                                        species1.getName(), species1OrganismPart1, species1OrganismPart1CellType1),
+                                species1OrganismPart1ExperimentAccession)),
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(
+                                        species1.getName(), species1OrganismPart1, species1OrganismPart1CellType2),
+                                species1OrganismPart1ExperimentAccession)),
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(
+                                        species1.getName(), species1OrganismPart1, species1OrganismPart1CellType3),
+                                species1OrganismPart1ExperimentAccession)),
+                        // Organism part 2, cell types
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(
+                                        species1.getName(), species1OrganismPart2, species1OrganismPart2CellType1),
+                                species1OrganismPart2ExperimentAccession)));
+    }
+
+    @DisplayName("This unit test use the randomised input test data and test the functionality of the wheel")
```

Comment:
Rephrase without words such as “This unit test...” or “This test...” (see below). This is more of a code comment rather than a test description, since it mentions implementation details. The `@DisplayName` should be a human-readable firendlier version of the test method; for instance, something like:
> Cell type wheel should not have the same value for the ID and its parent